### ISSUE TITLE
AirTable Integration

### ISSUE THREAD

**Issue by nellshamrell** [OperationCode/operationcode_backend#204]
# Bug Report
## What is the current behavior?

We appear to have code to automatically import new members to Airtable - [I've traced the code path](https://gist.github.com/nellshamrell/e0a1eb3e8b2cc0b13ab1c3bdb9457b6a).  However, I'm unsure if it is actually working - I'm creating this ticket to capture the work involved in verifying this and, if it's not working, fix it.

## What is the expected behavior?

When a user joins Operation Code, their names are sent to Airtable for tracking.

Once this functionality is verified, there are a few more steps to take.  The goal is to give our execs more insight into our membership without manually accessing the database console.

- [ ] Determine whether we need to create a new table/database/whatever Airtable calls them in Airtable itself. 
- [ ] If necessary, add a new table which tracks a user's first name, last name, slack name, service status (i.e. active or veteran or spouse), code school?, employed as a software developer?
- [ ] consider a Slack integration that sends a DM to each member requesting updates to their information - and if someone updates it, automatically update the record in Airtable.

**Comment by nellshamrell** [comment]
First step is learning the values for these secrets:

```
  def setup_airtable
    api_key  = OperationCode.fetch_secret_with(name: :airtable_api_key)
    base_id  = OperationCode.fetch_secret_with(name: :airtable_add_user_base_id)
    table_id = OperationCode.fetch_secret_with(name: :airtable_add_user_table_name)
    Operationcode::Airtable.new(api_key: api_key, base_id: base_id, table: table_id)
  end
```

which will tell me where the information is supposed to end up in Airtable, which will then allow me to verify whether it is actually there.

**Comment by nellshamrell** [comment]
Although we do have code to send info to Airtable - I don't think it actually ends up there.  I also don't think we have those secrets within our application (no one seems to know where they are if we do).

I'm going to proceed by creating a new Airtable...thingy to store member information - including Firstname, Lastname, Slack Name, Service Status, Employed as Software Developer?, and Code School Grad?

I'll then work on sending the info to Airtable for when a new member joins, then will focus on doing a mass export of the current member table into Airtable.  

@davidmolina has also requested we consider adding a slack bot that will DM our members each month requesting updates to the Service Status, Employed as Software Developer?, and Code School Grad? fields...I'll focus on that after getting the initial exporter working.

**Comment by nellshamrell** [comment]
Update:

I did some more digging into this:

I created a new Airtable base, with a table called Members, and with a new api key.  I set the relevant secrets, but it is failing mostly silently in inserting records into the base/table (it returns "false" as the result").  I took a look at the Gemfile, curious what kind of wrapper we have around the AirTable api - and it looks like we might have our own gem 'operationcode-airtable' - doing some research into this gem.
